charts/du/templates/statefulset.yaml

```
@@ -0,0 +1,88 @@
+{{- include
+  "accelleran.common.statefulset"
+  (fromYaml (include "accelleran.du.statefulset.args" .))
+-}}
+
+
+{{- define "accelleran.du.statefulset.args" -}}
+{{- $ := . -}}
+
+top:
+  {{ $ | toYaml | nindent 2 }}
+values:
+  {{ mergeOverwrite
+       (omit (deepCopy $.Values) "service")
+       (dict "service" (dict "ports" (($.Values).service).ports))
+     | toYaml | nindent 2
+  }}
+
+volumes:
+  - name: input-config
+    configMap:
+      name: {{ include "accelleran.common.fullname" (dict "top" $) }}-config
+  - name: output-config
+    emptyDir: {}
+  - name: hugepages
+    emptyDir:
+      medium: HugePages-1Gi
+
+volumeMounts:
+  - name: output-config
+    mountPath: /etc/config
+  - name: hugepages
+    mountPath: /dev/hugepages
+
+command:
+  - "accdu"
+  - "-c"
+  - "/etc/config/config.yml"
+
+initContainers:
+- {{ fromYaml (include "accelleran.common.init.container" (fromYaml (include "accelleran.du.init.config.args" .))) | toYaml | nindent 2 }}
+
+env:
+  - name: POD_IP
+    valueFrom:
+      fieldRef:
+        fieldPath: status.podIP
+{{- end -}}
+
+
+{{- define "accelleran.du.init.config.args" -}}
+{{- $image := $.Values.initImage -}}
+
+top:
+  {{ $ | toYaml | nindent 2 }}
+values:
+  {{
+    (mergeOverwrite
+      (deepCopy $.Values)
```

We don't need the privileges from the app container.

```suggestion
      (omit (deepCopy $.Values) "securityContext")
```